packages/react-ui/internal/ThemePlayground/__stories__/ThemePlayground.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { CSFStory } from 'creevey';
+
+import { ThemeContextPlayground } from '../ThemeContextPlayground';
+import { delay } from '../../../lib/utils';
+
+export default { title: 'ThemeProvider' };

```

Comment:
```suggestion
export default { title: 'ThemePlayground' };
```

Если идти до конца, то и родительскую историю стоит переименовать) Скриншоты тогда тоже переложить придется.